Sources/Kevin/Core/Styles/KevinTheme.swift

```diff
@@ -97,9 +97,27 @@ open class KevinTheme {
         public var shadowOffset = CGSize(width: 1.0, height: 1.0)
         public var shadowColor = UIColor(red: 1, green: 0, blue: 32/255, alpha: 1)
         
+        public var negativeBackgroundColor = UIColor(red: 230/255, green: 231/255, blue: 238/255, alpha: 1)
+        public var negativeTitleLabelTextColor = UIColor(red: 1, green: 0, blue: 32/255, alpha: 1)
+
         public init() { }
     }
-    
+        
+    public struct CardPaymentStyle {
+        public var titleFont = UIFont.systemFont(ofSize: 23, weight: .bold)
+        public var subtitleFont = UIFont.systemFont(ofSize: 14, weight: .regular)
+        public var textFieldTextColor = UIColor.black
+        public var textFieldFont = UIFont.systemFont(ofSize: 16, weight: .regular)
+        public var textFieldCornerRadius: CGFloat = 10
```

Comment:
Create TextFieldStyle and move everything below **textFieldTextColor**, titleFont and subtitleFont can be moved to the GeneralStyle I guess.